homeassistant/components/alexa/handlers.py

```diff
@@ -215,7 +238,7 @@ async def async_api_adjust_brightness(hass, config, directive, context):
     # read current state
     try:
         current = math.floor(
-            int(entity.attributes.get(light.ATTR_BRIGHTNESS)) / 255 * 100
+            get_attribute(directive, light.ATTR_BRIGHTNESS, int) / 255 * 100
```

Comment:
Shouldn't this have a default? Our brightness can be `None` and is valid in those cases.

```suggestion
            get_attribute(directive, light.ATTR_BRIGHTNESS, int, 0) / 255 * 100
```

Reply:
We should only get this request from Alexa if we indicated that the entity supports brightness. If it supports brightness, we should always have a value.

Reply:
That is not true, an entity can currently have an unknown brightness, eventhough it is supported